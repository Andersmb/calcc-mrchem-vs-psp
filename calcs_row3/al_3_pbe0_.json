{
  "input": {
    "molecule": {
      "cavity_coords": [
        {
          "center": [
            0.0,
            0.0,
            -0.1060175349004544
          ],
          "radius": 1.82
        },
        {
          "center": [
            0.0,
            2.5889246,
            1.4189913350995456
          ],
          "radius": 0.79
        },
        {
          "center": [
            0.0,
            -2.5889246,
            1.4189913350995456
          ],
          "radius": 0.79
        }
      ],
      "cavity_width": 0.2,
      "charge": 0,
      "coords": [
        {
          "atom": "al",
          "xyz": [
            0.0,
            0.0,
            -0.1060175349004544
          ]
        },
        {
          "atom": "h",
          "xyz": [
            0.0,
            2.5889246,
            1.4189913350995456
          ]
        },
        {
          "atom": "h",
          "xyz": [
            0.0,
            -2.5889246,
            1.4189913350995456
          ]
        }
      ],
      "multiplicity": 2
    },
    "mpi": {
      "bank_size": 1,
      "numerically_exact": false,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 9,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -5
    },
    "printer": {
      "file_name": "al_3_pbe0_.inp",
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 1e-06,
          "shared_memory": true
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 1e-06
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 1e-06,
          "shared_memory": true,
          "smooth_prec": 1e-06
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "pbe0"
              }
            ],
            "spin": true
          }
        }
      },
      "initial_guess": {
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": true,
        "method": "DFT (PBE0)",
        "prec": 0.0001,
        "relativity": "None",
        "restricted": false,
        "screen": 12.0,
        "type": "sad_gto",
        "zeta": 0
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 1e-06,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      },
      "scf_solver": {
        "checkpoint": false,
        "derivative": "abgv_55",
        "energy_thrs": 1e-06,
        "file_chk": "checkpoint/phi_scf",
        "final_prec": 1e-06,
        "helmholtz_prec": -1.0,
        "kain": 10,
        "light_speed": -1.0,
        "localize": true,
        "max_iter": 200,
        "method": "DFT (PBE0)",
        "orbital_thrs": 4.9999999999999996e-05,
        "proj_prec": 1e-06,
        "relativity": "None",
        "rotation": 0,
        "shared_memory": true,
        "smooth_prec": 1e-06,
        "start_prec": 1e-06
      },
      "write_orbitals": {
        "file_phi_a": "orbitals/phi_a_scf",
        "file_phi_b": "orbitals/phi_b_scf",
        "file_phi_p": "orbitals/phi_p_scf"
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        0.0,
        -1.0649663698192276e-18,
        1.380532195876737e-17
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 0.14577273084981174,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            1.1371625248642749e-05,
            1.5708806298420086e-05,
            -0.14577272955985698
          ],
          "vector_el": [
            1.1371541639478738e-05,
            1.5708733944297393e-05,
            -1.60552744598393
          ],
          "vector_nuc": [
            8.360916401104075e-11,
            7.235412269324115e-11,
            1.459754716424073
          ]
        }
      },
      "geometry": [
        {
          "symbol": "Al",
          "xyz": [
            0.0,
            0.0,
            -0.1060175349004544
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            0.0,
            2.5889246,
            1.4189913350995456
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            0.0,
            -2.5889246,
            1.4189913350995456
          ]
        }
      ],
      "multiplicity": 2,
      "orbital_energies": {
        "energy": [
          -56.162888533702024,
          -4.192443559814093,
          -2.725385072941765,
          -2.72244402813345,
          -2.717162285500517,
          -0.42501590786606025,
          -0.32213580343862197,
          -0.20535083122436454,
          -56.16026612984489,
          -4.1884210662076695,
          -2.7230052993711156,
          -2.7151203343209667,
          -2.7141066837552787,
          -0.40511068156002256,
          -0.3091115957294714
        ],
        "occupation": [
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0
        ],
        "spin": [
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b"
        ],
        "sum_occupied": -138.68796781341032
      },
      "scf_energy": {
        "E_ee": 122.53662435309424,
        "E_eext": 0.0,
        "E_el": -252.30014251904086,
        "E_en": -598.6457855504752,
        "E_kin": 243.0319466084184,
        "E_next": 0.0,
        "E_nn": 8.846259417077794,
        "E_nuc": 8.846259417077794,
        "E_tot": -243.45388310196307,
        "E_x": -4.700179444152322,
        "E_xc": -14.522748485925886,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 30,
      "nthreads": 6,
      "routine": "mrchem.x",
      "total_cores": 180,
      "version": "1.1.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 121.99371033303397,
        "E_eext": 0.0,
        "E_el": -250.92499221950297,
        "E_en": -594.7435763295747,
        "E_kin": 241.00133580892287,
        "E_next": 0.0,
        "E_nn": 8.846259417077794,
        "E_nuc": 8.846259417077794,
        "E_tot": -242.07873280242518,
        "E_x": -4.69159526576877,
        "E_xc": -14.48486676611635,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "scf_solver": {
        "converged": true,
        "cycles": [
          {
            "energy_terms": {
              "E_ee": 122.08808526142666,
              "E_eext": 0.0,
              "E_el": -252.24999144650434,
              "E_en": -595.6448557956104,
              "E_kin": 240.46583434384337,
              "E_next": 0.0,
              "E_nn": 8.846259417077794,
              "E_nuc": 8.846259417077794,
              "E_tot": -243.40373202942655,
              "E_x": -4.686192685961472,
              "E_xc": -14.47286257020253,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -243.40373202942655,
            "energy_update": 1.3249992270013706,
            "mo_residual": 0.16792274823171968,
            "wall_time": 91.942017413
          },
          {
            "energy_terms": {
              "E_ee": 122.27956885239331,
              "E_eext": 0.0,
              "E_el": -252.29187147153252,
              "E_en": -597.222045024086,
              "E_kin": 241.83724724536555,
              "E_next": 0.0,
              "E_nn": 8.846259417077794,
              "E_nuc": 8.846259417077794,
              "E_tot": -243.44561205445473,
              "E_x": -4.691944108389608,
              "E_xc": -14.494698436815781,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -243.44561205445473,
            "energy_update": 0.04188002502817767,
            "mo_residual": 0.04557023321009219,
            "wall_time": 64.810905767
          },
          {
            "energy_terms": {
              "E_ee": 122.40794292329952,
              "E_eext": 0.0,
              "E_el": -252.2979534724327,
              "E_en": -598.0060816881066,
              "E_kin": 242.5049751725783,
              "E_next": 0.0,
              "E_nn": 8.846259417077794,
              "E_nuc": 8.846259417077794,
              "E_tot": -243.45169405535492,
              "E_x": -4.695950253251711,
              "E_xc": -14.508839626952161,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -243.45169405535492,
            "energy_update": 0.006082000900192952,
            "mo_residual": 0.023123883269000577,
            "wall_time": 53.447305258
          },
          {
            "energy_terms": {
              "E_ee": 122.5226974870206,
              "E_eext": 0.0,
              "E_el": -252.2996097004856,
              "E_en": -598.6144730781175,
              "E_kin": 243.01280955054062,
              "E_next": 0.0,
              "E_nn": 8.846259417077794,
              "E_nuc": 8.846259417077794,
              "E_tot": -243.45335028340781,
              "E_x": -4.6995448097968975,
              "E_xc": -14.521098850132438,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -243.45335028340781,
            "energy_update": 0.0016562280528944484,
            "mo_residual": 0.022985827158524354,
            "wall_time": 54.752499836
          },
          {
            "energy_terms": {
              "E_ee": 122.56574451228244,
              "E_eext": 0.0,
              "E_el": -252.2999791337572,
              "E_en": -598.829934035983,
              "E_kin": 243.19088944032353,
              "E_next": 0.0,
              "E_nn": 8.846259417077794,
              "E_nuc": 8.846259417077794,
              "E_tot": -243.45371971667942,
              "E_x": -4.700945835534933,
              "E_xc": -14.52573321484524,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -243.45371971667942,
            "energy_update": 0.000369433271600883,
            "mo_residual": 0.014442576173051894,
            "wall_time": 56.888532452
          },
          {
            "energy_terms": {
              "E_ee": 122.5426925886148,
              "E_eext": 0.0,
              "E_el": -252.30007885152463,
              "E_en": -598.6722793309559,
              "E_kin": 243.05308975365043,
              "E_next": 0.0,
              "E_nn": 8.846259417077794,
              "E_nuc": 8.846259417077794,
              "E_tot": -243.45381943444684,
              "E_x": -4.700289492203518,
              "E_xc": -14.523292370630385,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -243.45381943444684,
            "energy_update": 9.971776742645488e-05,
            "mo_residual": 0.010421147718618955,
            "wall_time": 59.001037384
          },
          {
            "energy_terms": {
              "E_ee": 122.5342172653192,
              "E_eext": 0.0,
              "E_el": -252.30010586325386,
              "E_en": -598.5955280304426,
              "E_kin": 242.9836479482652,
              "E_next": 0.0,
              "E_nn": 8.846259417077794,
              "E_nuc": 8.846259417077794,
              "E_tot": -243.45384644617607,
              "E_x": -4.700112982443348,
              "E_xc": -14.52233006395239,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -243.45384644617607,
            "energy_update": 2.7011729230252968e-05,
            "mo_residual": 0.006929686991946493,
            "wall_time": 58.911709224
          },
          {
            "energy_terms": {
              "E_ee": 122.52882992000777,
              "E_eext": 0.0,
              "E_el": -252.30012624893794,
              "E_en": -598.5976096353861,
              "E_kin": 242.9906678646299,
              "E_next": 0.0,
              "E_nn": 8.846259417077794,
              "E_nuc": 8.846259417077794,
              "E_tot": -243.45386683186015,
              "E_x": -4.700015439066123,
              "E_xc": -14.521998959123385,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -243.45386683186015,
            "energy_update": 2.0385684081247746e-05,
            "mo_residual": 0.00818469639957998,
            "wall_time": 57.780114263
          },
          {
            "energy_terms": {
              "E_ee": 122.5300780335727,
              "E_eext": 0.0,
              "E_el": -252.30013439171515,
              "E_en": -598.623616699948,
              "E_kin": 243.01569634869915,
              "E_next": 0.0,
              "E_nn": 8.846259417077794,
              "E_nuc": 8.846259417077794,
              "E_tot": -243.45387497463736,
              "E_x": -4.700052748267612,
              "E_xc": -14.522239325771414,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -243.45387497463736,
            "energy_update": 8.14277720451173e-06,
            "mo_residual": 0.0035848023759854457,
            "wall_time": 58.703920022
          },
          {
            "energy_terms": {
              "E_ee": 122.53734680215331,
              "E_eext": 0.0,
              "E_el": -252.3001404758193,
              "E_en": -598.6602252226527,
              "E_kin": 243.04592138974107,
              "E_next": 0.0,
              "E_nn": 8.846259417077794,
              "E_nuc": 8.846259417077794,
              "E_tot": -243.4538810587415,
              "E_x": -4.700243930507697,
              "E_xc": -14.522939514553281,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -243.4538810587415,
            "energy_update": 6.084104143155855e-06,
            "mo_residual": 0.0029429287236853703,
            "wall_time": 60.398521588
          },
          {
            "energy_terms": {
              "E_ee": 122.53873344412246,
              "E_eext": 0.0,
              "E_el": -252.30014151111632,
              "E_en": -598.6605520174168,
              "E_kin": 243.0448903586115,
              "E_next": 0.0,
              "E_nn": 8.846259417077794,
              "E_nuc": 8.846259417077794,
              "E_tot": -243.45388209403853,
              "E_x": -4.7002366004080365,
              "E_xc": -14.522976696025454,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -243.45388209403853,
            "energy_update": 1.0352970321036992e-06,
            "mo_residual": 0.0012350819261324419,
            "wall_time": 56.551375791
          },
          {
            "energy_terms": {
              "E_ee": 122.53748559267093,
              "E_eext": 0.0,
              "E_el": -252.30014214892765,
              "E_en": -598.6454047568084,
              "E_kin": 243.03071540258145,
              "E_next": 0.0,
              "E_nn": 8.846259417077794,
              "E_nuc": 8.846259417077794,
              "E_tot": -243.45388273184986,
              "E_x": -4.700173029048685,
              "E_xc": -14.522765358322935,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -243.45388273184986,
            "energy_update": 6.378113255323115e-07,
            "mo_residual": 0.0008679231729335374,
            "wall_time": 55.132355652
          },
          {
            "energy_terms": {
              "E_ee": 122.53643852448116,
              "E_eext": 0.0,
              "E_el": -252.30014238955755,
              "E_en": -598.6394546644854,
              "E_kin": 243.025694873907,
              "E_next": 0.0,
              "E_nn": 8.846259417077794,
              "E_nuc": 8.846259417077794,
              "E_tot": -243.45388297247976,
              "E_x": -4.700151947794441,
              "E_xc": -14.52266917566591,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -243.45388297247976,
            "energy_update": 2.4062990178208565e-07,
            "mo_residual": 0.0004530088061683549,
            "wall_time": 55.326761712
          },
          {
            "energy_terms": {
              "E_ee": 122.53619293380767,
              "E_eext": 0.0,
              "E_el": -252.30014248036062,
              "E_en": -598.6413153240801,
              "E_kin": 243.02781321386644,
              "E_next": 0.0,
              "E_nn": 8.846259417077794,
              "E_nuc": 8.846259417077794,
              "E_tot": -243.45388306328283,
              "E_x": -4.7001561533974066,
              "E_xc": -14.522677150557257,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -243.45388306328283,
            "energy_update": 9.080306995201681e-08,
            "mo_residual": 0.00022648173407194226,
            "wall_time": 55.992446721
          },
          {
            "energy_terms": {
              "E_ee": 122.53632049347185,
              "E_eext": 0.0,
              "E_el": -252.3001425096705,
              "E_en": -598.64391535428,
              "E_kin": 243.03033457062156,
              "E_next": 0.0,
              "E_nn": 8.846259417077794,
              "E_nuc": 8.846259417077794,
              "E_tot": -243.45388309259272,
              "E_x": -4.700168934776892,
              "E_xc": -14.52271328470704,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -243.45388309259272,
            "energy_update": 2.9309887850104133e-08,
            "mo_residual": 0.00012396038001644515,
            "wall_time": 55.375338483
          },
          {
            "energy_terms": {
              "E_ee": 122.53662435309424,
              "E_eext": 0.0,
              "E_el": -252.30014251904086,
              "E_en": -598.6457855504752,
              "E_kin": 243.0319466084184,
              "E_next": 0.0,
              "E_nn": 8.846259417077794,
              "E_nuc": 8.846259417077794,
              "E_tot": -243.45388310196307,
              "E_x": -4.700179444152322,
              "E_xc": -14.522748485925886,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -243.45388310196307,
            "energy_update": 9.370353382109897e-09,
            "mo_residual": 7.030208542572133e-05,
            "wall_time": 65.02454251
          }
        ],
        "wall_time": 960.088396511
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
