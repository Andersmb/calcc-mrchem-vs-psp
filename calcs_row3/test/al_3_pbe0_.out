                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Al            :        0.000000        0.000000       -0.106018
    1       H            :        0.000000        2.588925        1.418991
    2       H            :        0.000000       -2.588925        1.418991
---------------------------------------------------------------------------
 Center of mass          :        0.000000       -0.000000        0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               8
 Beta electrons          :                                               7
 Total electrons         :                                              15
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999999835283e-01
    1    1    a          :                              9.999999258923e-01
    2    1    a          :                              1.000000051403e+00
    3    1    a          :                              9.999999801410e-01
    4    1    a          :                              9.999999886048e-01
    5    1    a          :                              1.000000855327e+00
    6    1    a          :                              9.999968260921e-01
    7    1    a          :                              1.000001120692e+00
    8    1    b          :                              9.999999835283e-01
    9    1    b          :                              9.999999258923e-01
   10    1    b          :                              1.000000051403e+00
   11    1    b          :                              9.999999801410e-01
   12    1    b          :                              9.999999886048e-01
   13    1    b          :                              1.000000855327e+00
   14    1    b          :                              9.999968261028e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                241.001335808923
 E-N energy              :            (au)               -594.743576329575
 Coulomb energy          :            (au)                121.993710333034
 Exchange energy         :            (au)                 -4.691595265769
 X-C energy              :            (au)                -14.484866766116
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  8.846259417078
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -250.924992219503
 Nuclear energy          :            (au)                  8.846259417078
---------------------------------------------------------------------------
 Total energy            :            (au)             -2.420787328024e+02
                         :      (kcal/mol)             -1.519066970770e+05
                         :        (kJ/mol)             -6.355776209828e+05
                         :            (eV)             -6.587297610703e+03
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -55.805843006904
    1    1    a          :            (au)                 -4.184471083367
    2    1    a          :            (au)                 -2.713326545691
    3    1    a          :            (au)                 -2.710867006406
    4    1    a          :            (au)                 -2.706346311891
    5    1    a          :            (au)                 -0.421653613500
    6    1    a          :            (au)                 -0.306207793365
    7    1    a          :            (au)                 -0.200035917793
    8    1    b          :            (au)                -55.803085024036
    9    1    b          :            (au)                 -4.179698417286
   10    1    b          :            (au)                 -2.710443863376
   11    1    b          :            (au)                 -2.703801395364
   12    1    b          :            (au)                 -2.702423809202
   13    1    b          :            (au)                 -0.398893567268
   14    1    b          :            (au)                 -0.289956461991
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -137.837053817439
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          3.872983e+00        -242.078732802425         -2.420787e+02
    1          1.679227e-01        -243.403732029427         -1.324999e+00
    2          4.557023e-02        -243.445612054455         -4.188003e-02
    3          2.312388e-02        -243.451694055355         -6.082001e-03
    4          2.298583e-02        -243.453350283408         -1.656228e-03
    5          1.444258e-02        -243.453719716679         -3.694333e-04
    6          1.042115e-02        -243.453819434447         -9.971777e-05
    7          6.929687e-03        -243.453846446176         -2.701173e-05
    8          8.184696e-03        -243.453866831860         -2.038568e-05
    9          3.584802e-03        -243.453874974637         -8.142777e-06
   10          2.942929e-03        -243.453881058742         -6.084104e-06
   11          1.235082e-03        -243.453882094039         -1.035297e-06
   12          8.679232e-04        -243.453882731850         -6.378113e-07
   13          4.530088e-04        -243.453882972480         -2.406299e-07
   14          2.264817e-04        -243.453883063283         -9.080307e-08
   15          1.239604e-04        -243.453883092593         -2.930989e-08
   16          7.030209e-05        -243.453883101963         -9.370353e-09
---------------------------------------------------------------------------
                      SCF converged in 16 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Al            :        0.000000        0.000000       -0.106018
    1       H            :        0.000000        2.588925        1.418991
    2       H            :        0.000000       -2.588925        1.418991
---------------------------------------------------------------------------
 Center of mass          :        0.000000       -0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                243.031946608418
 E-N energy              :            (au)               -598.645785550475
 Coulomb energy          :            (au)                122.536624353094
 Exchange energy         :            (au)                 -4.700179444152
 X-C energy              :            (au)                -14.522748485926
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  8.846259417078
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -252.300142519041
 Nuclear energy          :            (au)                  8.846259417078
---------------------------------------------------------------------------
 Total energy            :            (au)             -2.434538831020e+02
                         :      (kcal/mol)             -1.527696169113e+05
                         :        (kJ/mol)             -6.391880775717e+05
                         :            (eV)             -6.624717355005e+03
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -56.162888533702
    1    1    a          :            (au)                 -4.192443559814
    2    1    a          :            (au)                 -2.725385072942
    3    1    a          :            (au)                 -2.722444028133
    4    1    a          :            (au)                 -2.717162285501
    5    1    a          :            (au)                 -0.425015907866
    6    1    a          :            (au)                 -0.322135803439
    7    1    a          :            (au)                 -0.205350831224
    8    1    b          :            (au)                -56.160266129845
    9    1    b          :            (au)                 -4.188421066208
   10    1    b          :            (au)                 -2.723005299371
   11    1    b          :            (au)                 -2.715120334321
   12    1    b          :            (au)                 -2.714106683755
   13    1    b          :            (au)                 -0.405110681560
   14    1    b          :            (au)                 -0.309111595729
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -138.687967813410
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.000011        0.000016       -1.605527
 Magnitude               :            (au)                        1.605527
                         :         (Debye)                        4.080843
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000        1.459755
 Magnitude               :            (au)                        1.459755
                         :         (Debye)                        3.710326
---------------------------------------------------------------------------
 Total vector            :        0.000011        0.000016       -0.145773
 Magnitude               :            (au)                        0.145773
                         :         (Debye)                        0.370517
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 17m  5s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
